screwdriver.yaml

```diff
@@ -2,7 +2,7 @@ workflow:
     - deploy
 
 shared:
-    image: golang
+    image: golang:1.8.3
```

Comment:
Pin this due to the issue here: https://github.com/screwdriver-cd/screwdriver/issues/729